app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt

```diff
@@ -80,6 +81,14 @@ fun BitwardenActionCard(
                     modifier = Modifier.offset(x = 8.dp),
                 )
             }
+            if (cardSubtitle != null) {
+                Spacer(Modifier.height(4.dp))
+                Text(
+                    text = cardSubtitle,
+                    style = BitwardenTheme.typography.bodyMedium,
+                    color = BitwardenTheme.colorScheme.text.primary,
```

Comment:
I believe the `bitwardenCardColors` should be handling this, can we double check?

Reply:
nice!

Reply:
confirmed